### ISSUE TITLE
Lookup dropdown in MaterialTable jumps on scrolling

### ISSUE THREAD

**Issue by rnsangos** [mbrn/material-table#2936]
Hi,

I am creating a inventory table, which has a drop down to select from. I am using MaterialUI. The lookup box itself jumps as the selection changes. It looks like it is redrawing the box based on the selection. I tried putting the MenuProps block as I read it would resolve the Scroll jumping issue. Still no luck. The whole code is in the below link for you to reproduce the issue. 
Please suggest.

  [CodeSandbox](https://codesandbox.io/s/material-table-starter-template-forked-gfn8n?file=/src/index.js)
 
Best Regards
Sangeetha

**Comment by dogfootruler-kr** [comment]
Hello,

I don't know if you found how to solve this but I stumble upon the same issue and this is not a MaterialTable bug but more of an option from the MUI library.

You can find the fix there : https://stackoverflow.com/a/59790471

TLDR: Create a custom filterComponent and use this MenuProps for you Select component
```
const ITEM_HEIGHT = 48;
const ITEM_PADDING_TOP = 8;
const MenuProps = {
  PaperProps: {
    style: {
      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
      width: 200,
    },
  },
  variant: "menu",
  getContentAnchorEl: null
};
<Select MenuProps={MenuProps}></Select>
